src/fix_wall.cpp

```diff
@@ -44,6 +42,7 @@ FixWall::FixWall(LAMMPS *lmp, int narg, char **arg) : Fix(lmp, narg, arg), nwall
   virial_global_flag = virial_peratom_flag = 1;
   respa_level_support = 1;
   ilevel_respa = 0;
+  ewall = new double[7];
```

Comment:
Should not be necessary.

Reply:
fixed by commit https://github.com/lammps/lammps/commit/cac6b5dc02afcff7b42c96a90cf829a4da537a2a